packages/core/src/components/graphs/radar.ts

```diff
@@ -444,6 +444,22 @@ export class Radar extends Component {
 									c
 								).y
 						)
+						.end()
+						.finally(() => {
+							// Align chart horizontally after x-axies has finished rendering
+							const alignment = Tools.getProperty(
+								options,
+								'radar',
+								'alignment'
+							);
+
+							const alignmentXOffset = this.getAlignmentXOffset(
+								alignment,
+								svg,
+								this.getParent()
+							);
+							svg.attr('x', alignmentXOffset);
+						})
```

Comment:
this is running after .update(), what about .enter()?

Reply:
.enter() will be called only once - when the radar chart is being created. Hence, it makes more sense to add it to update since the element exists and the position can be realigned when any changes are made.

I cannot chain the promise to the .call function. If I chain it inside the call function, then the alignment will be incorrect on fullscreen exit. This is the same case if I were to add the above code to **both** enter and update.